utils/orca_proxy/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nginx:1.18.0-alpine
+
+COPY nginx.conf /etc/nginx/nginx.conf
+
+CMD ["nginx"]
```

Comment:
I still can't see the empty line at the end of this file 😉

Reply:
I might have missed that :) Fixed now